src/main/kotlin/org/blueliner/springbootwithkotlin/dto/request/OpenAiRequest.kt

```diff
@@ -0,0 +1,9 @@
+package org.blueliner.springbootwithkotlin.dto.request
+
+data class OpenAiRequest (
+    val model: String,
+    val max_tokens: Int,
```

Comment:
use json camel case serialization strategy, underscores in field namings are bad practice